src/scripts/components/MultiAvatar.jsx

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+class MultiAvatar extends React.Component {
+  renderInitials(initials) {
+    return (
+      <text x="50" y="50" textAnchor="middle" dy="0.35em" fontSize="30">{initials}</text>
+    );
+  }
+
+  formatInitials(name) {
+    const splitName = name.split(' ');
+    return splitName[0][0] + splitName[splitName.length - 1][0];
+  }
+
+  renderAvatars() {
+    const { foregroundImageUrl, backgroundImageUrl, foregroundName, backgroundName } = this.props;
+
+    const foregroundInitials = this.formatInitials(foregroundName);
+    const backgroundInitials = this.formatInitials(backgroundName);
+
+    const foregroundImage = foregroundImageUrl;
+    const backgroundImage = backgroundImageUrl;
+
+    return (
+      <div className="multi-avatars">
+        <svg
+          viewBox="0 0 100 100"
+          xmlns="http://www.w3.org/2000/svg"
+          preserveAspectRatio="none"
+          className="avatar avatar--member multi-avatars__background-avatar"
+          style={backgroundImage ? { backgroundImage: `url(${backgroundImage})` } : {}}
+        >
+          {backgroundImage === '' && this.renderInitials(backgroundInitials)}
+        </svg>
+        <svg
+          viewBox="0 0 100 100"
+          xmlns="http://www.w3.org/2000/svg"
+          preserveAspectRatio="none"
+          className="avatar avatar--member multi-avatars__foreground-avatar"
+          style={foregroundImage ? { backgroundImage: `url(${foregroundImage})` } : {}}
+        >
+          {foregroundImage === '' && this.renderInitials(foregroundInitials)}
+        </svg>
+      </div>
+    );
+  }
+
+  render() {
```

Comment:
Minor thing, you probably can just put everything in the render method. render functions are great when there's lots of JSX blocks that crowd the main render function, in this case we're just calling that one block. This is personal preference so just do what you prefer here.

Reply:
+1 on this. Personal preference but I try and put most of the JSX in there.

Reply:
Seems fine to me! Updated.